docs/source/settings.rst

```diff
@@ -428,6 +428,14 @@ to enable or disable its usage.
 .. versionchanged:: 19.6
    added support for the ``SENDFILE`` environment variable
 
+.. _reuse-port:
+
+reuse_port
+~~~~~~~~~~
+
+* ``--reuse-port``
```

Comment:
Documentation for the new option will be generated automatically from `gunicorn/config.py` when you run `make html` in `docs/` directory.

Reply:
I'm not sure I understand your comment, @berkerpeksag. I generated this change by running `` make -C docs html``. Am I not supposed to commit settings.rst?

Reply:
I thought you manually edited it because I didn't see the "Set the ``SO_REUSEPORT`` flag on the listening socket." part in the generated file. Now I don't know why `make html` didn't get the documentation in the `desc` attribute.

Reply:
I generated it, but then edited it because running it also brought in ``logconfig_dict`` (which presumably was omitted previously and should be in its own PR), and also changed ``chdir``, ``user`` and ``group``. I must have got the edit wrong. Sorry about that. It's all there now.